demo_jupyterhub_conf.py

```diff
@@ -0,0 +1,74 @@
+# Sample jupyterhub configuration using jupyterhub_moss
+
+import os
+import random
+import sys
+import batchspawner  # noqa
+
+sys.path.insert(0, os.path.abspath(os.path.dirname(__file__)))
+
+import jupyterhub_moss  # noqa
+
+c = get_config()  # noqa
+
+# Select the MOSlurmSpawner backend and increase the timeout since batch jobs may take time to start
+jupyterhub_moss.set_config(c)
+
+
+class MOckSlurmSpawner(jupyterhub_moss.MOSlurmSpawner):
+    def _get_slurm_info(self):
+        return {
+            k: {"nodes": v["max_nprocs"], "idle": random.randint(0, v["max_nprocs"])}
+            for k, v in self.partitions.items()
+        }
+
+
+c.JupyterHub.spawner_class = MOckSlurmSpawner
+
+# Partition descriptions
+c.MOSlurmSpawner.partitions = {
+    "partition_1": {  # Partition name     # (See description of fields below for more info)
```

Comment:
I would remove these comments with a link to https://github.com/silx-kit/jupyterhub_moss#partition-settings instead